CMakeLists.txt

```diff
@@ -2219,29 +2217,49 @@ if(FAAD)
   endif()
 endif()
 
+# FDK-AAC is loaded dynamically at runtime by EncoderFdkAac using QLibrary,
+# so copy it into the Windows and macOS packages, but do not link to it.
+find_library(FDK_AAC_LIBRARY fdk-aac)
+# On Windows find_library finds the .lib file, but the installer needs the .dll file.
+find_file(FDK_AAC_DLL fdk-aac.dll PATH_SUFFIXES ${CMAKE_INSTALL_BINDIR})
+if(APPLE AND MACOS_BUNDLE AND FDK_AAC_LIBRARY)
+  message(STATUS "Found fdk-aac: ${FDK_AAC_LIBRARY}")
+  file(COPY ${FDK_AAC_LIBRARY}
+    DESTINATION ${MIXXX_INSTALL_PREFIX}/Contents/Frameworks
+    FOLLOW_SYMLINK_CHAIN
+  )
+elseif(WIN32 AND FDK_AAC_DLL)
+  message(STATUS "Found fdk-aac DLL: ${FDK_AAC_DLL}")
+  install(FILES ${FDK_AAC_DLL} DESTINATION ${MIXXX_INSTALL_BINDIR})
+elseif(FDK_AAC_LIBRARY)
+  message(STATUS "Found fdk-aac: ${FDK_AAC_LIBRARY}")
+else()
+  message(STATUS "Could NOT find fdk-aac")
```

Comment:
This should be rephrased and give more target specific hints.

Reply:
Such as?

Reply:
Such as: "Could not find fdk-aac.dll in ${CMAKE_INSTALL_BINDIR}"

Reply:
I don't think that extra complexity is necessary or really helpful...

Reply:
Why not just add it? The fast merge of this PR is so important - please don't postpone it uneccessary by such detail discussions!

Reply:
I agree that merging this PR should not be slowed down by bikeshedding pointless details.